kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProxyIngressReconciler.java

```diff
@@ -71,9 +71,10 @@ public UpdateControl<KafkaProxyIngress> reconcile(
 
         KafkaProxyIngress patch;
         if (proxyOpt.isPresent()) {
+            String checksum = MetadataChecksumGenerator.checksumFor(proxyOpt.get());
             patch = statusFactory.newTrueConditionStatusPatch(
                     ingress,
-                    Condition.Type.ResolvedRefs);
+                    Condition.Type.ResolvedRefs, checksum);
```

Comment:
How's this going to work?   The patch will patch both metadata and status, but we are applying it using ` UpdateControl.patchStatus` - which will just patch the status.

Reply:
Hmm, need to look at that. The unit test is validating that both patch and resource are set, ahh but its not validating `update.isPatchResource() == true`